frontend/src/hooks/queryBuilder/useAutoComplete.ts

```diff
@@ -130,5 +145,7 @@ export const useAutoComplete = (query: IBuilderQuery): IAutoComplete => {
 		searchValue,
 		isMulti,
```

Comment:
There is some `handleKeyDown` logic above. It's not clear what that is doing. If I use the arrow key up-down for selecting a value for `IN`, it instead makes a chip. It doesn't deselect but makes a chip. There must be some way to handle this. How do others do this?